website/versioned_docs/version-1.11.0/operator-docs/cumulus-dependencies.md

```diff
@@ -0,0 +1,73 @@
+---
+id: version-1.11.0-cumulus-dependencies
+title: Cumulus Dependencies
+hide_title: true
+original_id: cumulus-dependencies
+---
+
+# Updating Cumulus Dependencies
+
+
+## Cumulus Dependency Management - A Quick Intro to package.json
+
+As is standard with nodejs projects, Cumulus has lists of dependencies in `package.json` files. For this example, we'll look at the `package.json` file under the `example` directory in the main [Cumulus project](github.com/nasa/cumulus).
+
+In the `package.json` file, both the `dev_dependencies` and `dependencies` attributes should exist. The `dev_dependencies` are packages that are used in development, but are not necessary for deployment/usage of the package; a good example of this would be testing frameworks (`ava`) or mocking packages. The `dependencies` attribute lists all the required packages for using the software. Cumulus core developed dependencies all are prefixed with `@cumulus/` (i.e. the ingest package is listed as `@cumulus/ingest`).
+
+Example:
+```json
+...
+dependencies: {
+  "@cumulus/api": ^1.11.1,
```

Comment:
Same parse error as above

Reply:
Those are all just copies of the file you reviewed earlier. I'm going to have to make changes and then copy them through.